espnet/bin/mt_recog.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# Copyright 2019 Kyoto University (Hirofumi Inaguma)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+
+import configargparse
+import logging
+import os
+import random
+import sys
+
+import numpy as np
+
+
+def main(args):
+    parser = configargparse.ArgumentParser(
```

Comment:
Recently, we have changed the `parser` style to generate documentation. Define `get_parser() -> ArgumentParser` like other tools https://github.com/espnet/espnet/blob/7b0f759b459ec90131f9f1c915d9fa5677a1fa0a/espnet/bin/asr_train.py#L22